Sources/Shared/UI/Social/MessageComposerViewController.m

```diff
@@ -361,18 +368,18 @@ - (IBAction)onBtnSend:(id)sender
                 //release the date formatter because, not needed after that piece of code
                 [dateFormatter release];
 
-                fileAttachName = [NSString stringWithFormat:@"MobileImage_%@.png", fileAttachName];
+                fileAttachName = [NSString stringWithFormat:@"mobile_image_%@.png", fileAttachName];
                 fileAttachName = [fileAttachName stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
                 NSLog(@"uploading file: %@",fileAttachName);
                 
-                fileAttachURL = [NSString stringWithFormat:@"%@/Public/Mobile/%@", fileProxy._strUserRepository, fileAttachName];
+                fileAttachURL = [NSString stringWithFormat:@"%@/Mobile/%@",photosFolderPath,fileAttachName];
```

Comment:
Following the comment above, it would become:
`[NSString stringWithFormat:@"%@/%@/%@", photosFolderPath, MOBILE_UPLOAD_DEST_FOLDER, fileAttachName];`